packages/terra-functional-testing/src/commands/validates/screenshot.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * The screenshot comparisons.  It will capture screenshots of a specified element
+ * and assert the screenshot comparison results are within the mismatch tolerance or are an exact match
+ *
+ * This should be used within an `it` block.
+ *
+ * @param {[name, options]} [args] - the list of test arguments to parse.
+ * @param {string} [testName] - the required test case name.
+ * @param {Object} [options] - the visual regression test options. Options include mismatchTolerance and selector
+ * @param {Object} [options.misMatchTolerance] - the mismatch tolerance for the screenshot comparison.
+ * @param {string} [options.selector=global.Terra.serviceOptions.selector] - the element selector to use for
+ */
+const screenshot = (testName, options = {}) => {
+  if (!testName || typeof testName !== 'string' || testName.length === 0) {
+    throw new Error('[terra-functional-testing:screenshot] Terra.validate.screenshot requires a unique test name as the first argument.');
+  }
+
+  const { selector } = options;
+  const wrappedOptions = {
+    name: testName,
+    ...options,
```

Comment:
would this only be misMatchTolerance? passed in `....options`?

Reply:
It can be `mismatchTolerance` and/or `selector`: https://github.com/cerner/terra-toolkit/pull/545/files#diff-0aa7dac6eaa779fb29a0b1ac99068b18cbf2c0a6ecbc1db503237212266a8f68R23

In wdio4 we used to also allow an array of viewports but we decided to no longer support that.

Reply:
ahh I was looking at the code below on line 24 which passes the selector directly to checkElement